xactions.py

```diff
@@ -997,7 +997,7 @@ class _vi_dd(ViTextCommandBase):
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
 
-    def run(self, edit, mode=None, count=1, register='"'):
```

Comment:
Ok, I missed this comment for a while. It's not a big deal. The testing result shows that both register 1 and " are saved with the deleted lines after the modification. So it seems no impact to register ".